packages/bech32/src/decode.ts

```diff
@@ -0,0 +1,12 @@
+import { BECH32_ADDRESS_LENGTH } from './penumbra-bech32';
+import { bech32m } from 'bech32';
+
+export const bech32ToAddress143 = (addr: string): { inner: Uint8Array } => {
+  const decodeAddress = bech32m.decode(addr, BECH32_ADDRESS_LENGTH);
+  return { inner: new Uint8Array(bech32m.fromWords(decodeAddress.words)) };
+};
+
+export const bech32ToIdentityKey143 = (key: string): { ik: Uint8Array } => {
+  const decodeAddress = bech32m.decode(key, BECH32_ADDRESS_LENGTH);
+  return { ik: new Uint8Array(bech32m.fromWords(decodeAddress.words)) };
+};
```

Comment:
these are now more specific. the previous 'bech32ToUint8Array' also truncated at an arbitrary length, but now this makes it hopefully more obvious. is 143 correct for the identity key length?

Reply:
Does it clarify things to have `143` in the name? What does that inform to the developer? I think we probably drop unless there's another `bech32ToIdentityKey` version that we need to distinguish this from.

Reply:
143 characters is the expected address length. if the bech32 lib doesn't receive an arg about length, it truncates at 90, because this is the expected length of a bitcoin address. so any helper function we provide will always truncate at some specific length.

the previous 'bech32ToUint8Array' seemed named to indicate it could handle an input of arbitrary length, but it was written to truncate at 143, the length of a penumbra address.inner. but was also used to decode "identity keys", and i don't know if truncation at 143 is the appropriate behavior for that.